akka-stream/src/main/scala/akka/stream/impl/fusing/GraphInterpreter.scala

```diff
@@ -351,10 +353,7 @@ final class GraphInterpreter(
         def reportStageError(e: Throwable): Unit = {
           if (activeStage == null) throw e
           else {
-            // TODO: Get error reporting back
-            //val stage = stages(activeStage.stageId)
-
-            log.error(e, "Error in stage [{}]: {}", activeStage, e.getMessage)
+            log.error(e, "Error in stage [{}]: {}", activeStage.originalStage.getOrElse(activeStage), e.getMessage)
```

Comment:
We pass the exception as first parameter as well, wont that lead to the stack trace being written and all? Actually maybe the getMessage is duplication even, @drewhk had removed and I reintroduced that after looking at the old logic.